6_visualize/src/combine_storm_frames.R

```diff
@@ -0,0 +1,12 @@
+combine_storm_frames <- function(gif_file, task_names=NULL) {
+
+  # run imageMagick convert to build a gif
+  if(is.null(task_names)) task_names <- '*'
+  png_files <- paste(sprintf('6_visualize/tmp/gif_frame_%s.png', task_names), collapse=',')
+  magick_command <- sprintf('magick convert -delay 8 -loop 0 %s %s', '6_visualize/tmp/gif_frame_*.png', gif_file)
```

Comment:
Might want to have `8` be a %s w/ a decent variable name (`loop_delay`?), since this is a duration of each frame, and we'll want to know what it means and that this is what we should change when we want the gif to be quicker/slower.

Reply:
yes, good idea. i'll get that in the next PR

Reply:
or `animation_speed`?

Reply:
Yes, delay or frame delay is better than what I suggested. That wasn’t a good option

Reply:
https://github.com/USGS-VIZLAB/vizstorm-GIF/pull/61